src/go/rpk/pkg/cli/redpanda/start_test.go

```diff
@@ -272,19 +270,19 @@ func TestStartCommand(t *testing.T) {
 			return fs.MkdirAll("/arbitrary/path", 0o755)
 		},
 		postCheck: func(fs afero.Fs, _ *redpanda.RedpandaArgs, st *testing.T) {
-			path := testConfigPath
-			p := &config.Params{ConfigPath: "/arbitrary/path/redpanda.yaml"} // In command execution this will be done by with ParamsFromCommand
-			conf, err := p.Load(fs)
+			path := testConfigFlag
+			p := &config.Params{ConfigFlag: "/arbitrary/path/redpanda.yaml"} // In command execution this will be done by with ParamsFromCommand
```

Comment:
This was removed in a later commit -- I'm not sure how easy it will be to split this commit and rebase/squash. lmk if you'd like me to try otherwise -- resolving.